app/views/contexts/summary/group.py

```diff
@@ -77,10 +93,62 @@ def _build_blocks(
                     ]
                 )
 
+            else:
+                section = schema.get_section(location.section_id)
+
+                summary_item = self.get_summary_item_for_list_for_section_summary(
+                    summary=section.get("summary", {}),
+                    list_name=block.get("for_list", {}),
+                )
+                if not summary_item:
+                    continue
+
+                if block["type"] in ["ListCollector"]:
```

Comment:
This if should be done eariler.

Reply:
Removed now as it is explicitly checked above 👍